ui/src/app/pages/workspace/workspace-wrapper/component.ts

```diff
@@ -94,8 +98,11 @@ export class WorkspaceWrapperComponent implements OnInit, OnDestroy {
             this.setSidebarState(false);
           }
         }));
-    this.subscriptions.push(routeConfigDataStore.subscribe(({minimizeChrome}) => {
-      this.displayNavBar = !minimizeChrome;
+    this.subscriptions.push(routeConfigDataStore.subscribe((data) => {
```

Comment:
I don't fully follow - if this is a React route, is there guaranteed to be either no routeConfigData event, or just an empty object?

Reply:
All routes under/that utilize this component will be guaranteed to have an empty data object. You will get an error in Angular if no data object is specified in the route config.